domain/infosync/info.go

```diff
@@ -149,6 +149,9 @@ func (is *InfoSyncer) init(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
+	if config.GetGlobalConfig().NoRegister {
```

Comment:
I think we'd better to use `config.GetGlobalConfig().NoRegister` as a argument to `GlobalInfoSyncerInit`. It is best to reduce the strong correlation between internal objects and  `config.GetGlobalConfig()`.